### ISSUE TITLE
create new project

### ISSUE THREAD

**Issue by vrozen** [usethesource/rascal-language-servers#79]
There's no 'create new Rascal project' option yet.
Opening existing Rascal projects, created using Eclipse, does work.

**Comment by DavyLandman** [comment]
I agree, that experience is nice, but I don't think there exist any "new project" wizard in vs code.

I think they assume you use the language own scafolding. Like yeoman for npm or maven archetype's. So I think we'll have to go for that route (and have a clear documentation that show how to use it).

**Comment by jurgenvinju** [comment]
we can simply add a command for creating a new Rascal project with a small template. That would go a long way in onboarding new programmers. Currently it's too hard to get started from scratch.

**Comment by DavyLandman** [comment]
if we could have that command run `mvn archetype`, it would also work for context outside vscode

**Comment by DavyLandman** [comment]
I think for maven archetypes you need a special kind of project: https://maven.apache.org/guides/mini/guide-creating-archetypes.html

and maybe we could get that published on maven central? so that the initial experience just works without any custom maven repos?

**Comment by jurgenvinju** [comment]
that sounds good. we'll make a small project called `rascal-maven-archetypes` and put it both on grand-central and usethesource.io and it will contain a pom.xml that refers to usethesource.io so people have access to our latest releases without a thought.

**Comment by jurgenvinju** [comment]
Notice that #248 is working to solve this proposal.

**Comment by DavyLandman** [comment]
the [`viewsWelcome` contribution](https://code.visualstudio.com/api/references/contribution-points#contributes.viewsWelcome) would allow us to add this feature in case there is no folder opened. the `"when": "workspaceFolderCount == 0"` clause should be used to get in that pane.
